wolfcrypt/test/test.c

```diff
@@ -19,6 +19,10 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
  */
 
+/*
+** refer to https://csrc.nist.gov/projects/hash-functions
```

Comment:
This comment is not helpful for the test program.
A reference to a document whose test vectors you used, if any, at the test vector, is useful.

Reply:
To be clear, the link is fine, but it needs to go next to the vectors (assuming you used them from there).